_posts/2020-05-06-floss-weekly.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Presto on FLOSS Weekly"
+author: Dain Sundstrom and Manfred Moser
+excerpt_separator: <!--more-->
+---
+
+Spreading the word about our project is an important task to grow the community
+around Presto. With a large, lively community we can ensure the success of
+Presto. Today we had the opportunity to talk about Presto on the long running
+open source podcast [FLOSS Weekly](https://twit.tv/shows/floss-weekly).
+
+<!--more-->
+
+[Randal Schwartz](http://www.stonehenge.com/merlyn/) was joined by his co-host
+[Simon Phipps](https://webmink.com/about/). We introduced Presto overall and
+talked about use cases of Presto and the problems it can solve. Both hosts, as
+well as the live audience, had some great questions and we did our best to
+answer them.
+
+We moved through the history of Presto, current users and usage, the community
+around the project and Dain talked about some of the upcoming improvements. In
```

Comment:
I'd add a comma after "project" for readability

Reply:
Done